release/release_gold.sh

```diff
@@ -36,9 +36,22 @@ function check_usage {
 	LIFERAY_COMMON_LOG_DIR="${_PROMOTION_DIR%/*}"
 }
 
+function check_supported_versions {
```

Comment:
@Wykenz, can you please place this function in the file according the alphabetic order?

I mean, it should be placed before the method `check_usage`